docs/RELEASE_NOTES.md

```diff
@@ -34,6 +34,12 @@
 [Javadoc](https://commercetools.github.io/commercetools-sync-java/v/1.5.0/) | 
 [Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/1.5.0)
 
+- 🐞 **Bug Fixes** (1)
+    - **Commons** -  Fixed a bug in the `CustomUpdateActionUtils#buildNewOrModifiedCustomFieldsUpdateActions` which caused to generate unnecessary update actions for null custom field values. [#428](https://github.com/commercetools/commercetools-sync-java/issues/428)
```

Comment:
```suggestion
    - **Commons** -  Fixed a bug in the `CustomUpdateActionUtils#buildNewOrModifiedCustomFieldsUpdateActions` which generated unnecessary update actions for null custom field values. [#428](https://github.com/commercetools/commercetools-sync-java/issues/428)
```

Reply:
`buildNewOrModifiedCustomFieldsUpdateActions` is not an exposed utility so the users are actually not interesed about it.

What they are interested about is the called method and any exposed functionality that uses this:

```suggestion
    - **Commons** -  Fixed a bug in the custom fields update actions builders which generated duplicated unnecessary update actions for `null` custom field values. This affected any sync module where the resource contained custom fields (i.e. Product Sync, Category Sync, CartDiscount Sync and Inventory Sync). It also affected any update actions building utility in which the resource/sub-resource contained custom fields.  [#428](https://github.com/commercetools/commercetools-sync-java/issues/428)
```

Reply:
21c9a77337f4dd48d0cf7d59466d8f8388947db9

Reply:
It seems you didn't address this correctly also 😢

Reply:
ac9311a